containers/release-candidate/src/daffodil-release-candidate

```diff
@@ -300,6 +301,17 @@ case $PROJECT_REPO in
     yarn package
     cp *.vsix $DAFFODIL_RELEASE_DIR/bin/
 
+    echo "Building Extension Docs..."
+    doc_name="Apache-Daffodil-Extension-for-Visual-Studio-Code-$VERSION.docx"
+    git clone https://github.com/apache/daffodil-vscode.wiki.git .wiki
+    pandoc -t docx -f markdown -o .wiki/$doc_name .wiki/*.md
```

Comment:
Does pandoc allow you to create a single HTML file from multiple markdown files?  That is, does the following work as well?

```shell
    echo "Building Extension Docs..."
    doc_name="Apache-Daffodil-Extension-for-Visual-Studio-Code-$VERSION.html"
    git clone https://github.com/apache/daffodil-vscode.wiki.git .wiki
    pandoc -t html -f markdown -o .wiki/$doc_name .wiki/*.md
```

Everyone has a web browser and can read a HTML file, but I'm not sure everyone has a word processor which can read a .docx file.  Are there any compelling reasons for picking docx as the format for reading documentation?

Reply:
Yeah it can do a single HTML file as well. The snippet you sent would be correct, I was thinking we wanted something to download but maybe it is better to keep it as HTML instead of trying to use pdf or markdown that would be fine by me. I will say though on the html instance some of the images are huge that would be the main thing that looks off. @scholarsmate It would be nice to here your thoughts as I believe you made the portable document issue (https://github.com/apache/daffodil-vscode/issues/380) for the vscode repo.

Reply:
Another thing to consider with doc/pdf is there are animated gifs in our wiki. I'm not sure how well that translates to document formats. It might be worth considering if those gifs provide value and if the documentation could be made to work with static images, but that's another issue.

Reply:
@Shanedell, the portable document requirement stems from a need to have access to the documentation on an air-gapped SharePoint site.